toolchain/lexer/tokenized_buffer.cpp

```diff
@@ -653,29 +653,29 @@ auto TokenizedBuffer::GetTokenText(Token token) const -> llvm::StringRef {
   }
 
   CARBON_CHECK(token_info.kind == TokenKind::Identifier())
-      << "Only identifiers have stored text!";
+      << "Only identifiers have stored text: " << token_info.kind.Name();
```

Comment:
The other ones are easy to understand (`The token must...: <token>`), but I'm not sure about this one. Maybe
```suggestion
      << "The token must be an identifier, only identifiers have stored text: " << token_info.kind.Name();
```

Reply:
When I have a chance, I'll probably fix this in the opposite direction: CHECK prints the comparison, and I think all the "token must be X" comments are more redundant than helpful.

Reply:
Noting, made the edit to let the comparison speak for itself.